src/setup_payload/java/BUILD.gn

```diff
@@ -14,27 +14,24 @@
 
 import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
-
-import("${build_root}/config/android_abi.gni")
 import("${chip_root}/build/chip/java/rules.gni")
 
-if (defined(build_java_matter_controller)) {
-  build_java_matter_controller = build_java_matter_controller
-} else {
-  build_java_matter_controller = false
-}
-
 shared_library("jni") {
   output_name = "libSetupPayloadParser"
+  if (build_java_matter_controller) {
+    include_dirs = java_matter_controller_dependent_paths
+    output_dir = "${root_out_dir}/lib/jni"
+  } else {
+    import("${build_root}/config/android_abi.gni")
```

Comment:
import needs to be set at the beginning of the GN file

Reply:
no, android_abi could not appear when compiling java

Reply:
Yes, I means we need to move this import statement to the beginning of the build file with build flag 'build_java_matter_controller'.

Reply:
done, thanks